### DISCUSSION TITLE
Bidding/ Investor list algorithm

### DISCUSSION THREAD

**Discussion by ndkazu** [Fair-Squares/fair-squares#221] [Q&A]
Is there a document describing the current Investor selection process/algorithm for an asset?
This is the most important part of the Fair Square project, therefore, I think that even temporary solutions should be properly documented.

**Comment by letodunc** [comment]
This documentation is needed for all functionalities of the Fair Square project. [Here](https://github.com/Fair-Squares/fair-squares/issues/123) is the description of the algorithm established by the team in the M3.

**Comment by ndkazu** [comment]
Diagram of the bidding pallet:
![alt text](https://github.com/ndkazu/FS_update_929/blob/main/bidding1.png?raw=true)

**Reply by ndkazu** [reply]
In the next step, I will try to detail `get_investor_distribution` and `get_common_investor_distribution`.
Please, point to the needed corrections and contribute when you can.
This is for the FairSquare documentation.
